HashSetVsBitSet/Program.fs

```diff
@@ -32,7 +36,8 @@ type BitSetTracker (jobCount, machineCount, operationCount: int) =
     member internal _.Values = buckets
     
     member _.Item
-        with get (jobId: int<JobId>, machineId: int<MachineId>, operationId: int<OperationId>) =
+    
+        with [<MethodImpl(MethodImplOptions.AggressiveInlining)>] get (jobId: int<JobId>, machineId: int<MachineId>, operationId: int<OperationId>) =
```

Comment:
IIRC the F# compiler drops MethodImpl attributes for some reason.

Reply:
This is why I created the `SetMany` method to force the inlining.